app/controllers/map_assets_controller.rb

```diff
@@ -0,0 +1,50 @@
+class MapAssetsController < ApplicationController
+  before_action :set_asset, only: %i[show edit update destroy]
+
+  def index
+    @map_assets = MapAsset.order(created_at: :desc)
+  end
+
+  def show; end
+
+  def new
+    @map_asset = MapAsset.new
+  end
+
+  def edit; end
+
+  def create
+    @map_asset = MapAsset.new(asset_params)
+    respond_to do |format|
+      if @map_asset.save
```

Comment:
what if you store the value from `@map_asset.save` in another variable and use that variable in `if` conditions. then you won't have to rewrite `format.html`

Reply:
Added if else condition in one respond to block